locales/ja/json.json

```diff
@@ -20,7 +20,7 @@
     "Actions": "アクション",
     "Add": "追加",
     "Add a new team member to your team, allowing them to collaborate with you.": "新しいチームメンバーを追加",
-    "Add additional security to your account using two factor authentication.": "セキュリティ強化のため二段階認証を追加",
+    "Add additional security to your account using two factor authentication.": "セキュリティ強化のため二要素認証を追加",
```

Comment:
```suggestion
    "Add additional security to your account using two factor authentication.": "セキュリティ強化のため二要素認証を追加。",
```

Reply:
@andrey-helldar 

Thank you for your review.
I thought there are several similar sentences that there is no `。` at end of the sentence.
Do you think that I have to fix all of them as you suggested?

Reply:
We determine the presence of full stops at the end of sentences by the source phrase for translation.

Since the `Add additional security to your account using two factor authentication.` clause ends with a full stop, we also use it.

The fact that in the previous version of the translation there was no point, this is my omission. Apparently, I missed this point during the code review.

In general, we try to stick to the same style of writing as in the original.

Reply:
@andrey-helldar 

I understood.
I am going to fix the it as soon as possible.

Reply:
If it helps, GitHub provides a quick way to commit changes as long as they're valid.

![image](https://user-images.githubusercontent.com/10347617/177164727-4d8175e2-60a9-47b3-9be5-52cac6be5462.png)

Reply:
Wow, that's cool feature that I've never used before.
Thank you for your information.
I appreciated it :)